varia/sass/blocks/blog-posts/_style.scss

```diff
@@ -141,16 +148,16 @@
 					.has-background:not(.has-background-background-color) &,
 					[class*="background-color"]:not(.has-background-background-color) &,
 					[style*="background-color"] & {
-						color: currentColor;
+						color: currentcolor;
```

Comment:
I think this is being changed by the linter and I'm afraid this will break the rule in certain browsers since I believe the camelcase is correct for the keyword.

Reply:
I did a quick check here https://caniuse.com/?search=currentColor. Both seems to be supported by all browsers.
However, I reverted them to be safe on not breaking the sites.

I would have to add a style-lint rule to avoid this in future changes.

Reply:
yeah, after further googling, it appears most modern browsers will accept both, and we are no longer supporting IE so maybe we can ignore this